lensless/admm.py

```diff
@@ -130,7 +130,7 @@ def reset(self):
         if self.is_torch:
             # TODO initialize without padding
             if self._image_est is None:
-                self._image_est = torch.zeros(self._padded_shape, dtype=self._dtype).to(
+                self._image_est = torch.zeros([1] + self._padded_shape, dtype=self._dtype).to(
```

Comment:
singleton dimension added for batch size? probably worth adding a comment why this is done